rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/reporting/rules/CreateAboutWindupReportRuleProvider.java

```diff
@@ -0,0 +1,105 @@
+{
+    public static final String REPORT_NAME = "About";
+    public static final String TEMPLATE_APPLICATION_REPORT = "/reports/templates/about_windup.ftl";
+
+    public CreateAboutWindupReportRuleProvider()
+    {
+        super(MetadataBuilder.forProvider(CreateAboutWindupReportRuleProvider.class)
+                    .setPhase(ReportGenerationPhase.class));
+    }
+
+    // @formatter:off
+    @Override
+    public Configuration getConfiguration(GraphContext context)
+    {
+        ConditionBuilder applicationProjectModelsFound = Query
+                    .fromType(WindupConfigurationModel.class);
+
+        AbstractIterationOperation<WindupConfigurationModel> addApplicationReport = new AbstractIterationOperation<WindupConfigurationModel>()
+        {
+            @Override
+            public void perform(GraphRewrite event, EvaluationContext context, WindupConfigurationModel payload)
+            {
+                ProjectModel projectModel = payload.getInputPath().getProjectModel();
+                if (projectModel == null)
+                {
+                    throw new WindupException("Error, no project found in: " + payload.getInputPath().getFilePath());
+                }
+                createApplicationReport(event.getGraphContext(), projectModel);
+            }
+            
+            @Override
+            public String toString()
+            {
+                return "CreateAboutWindupReport";
+            }
+        };
+
+        return ConfigurationBuilder.begin()
+                    .addRule()
+                    .when(applicationProjectModelsFound)
+                    .perform(addApplicationReport);
+
+    }
+    // @formatter:on
+
+    private ApplicationReportModel createApplicationReport(GraphContext context, ProjectModel projectModel)
+    {
+    	ApplicationReportService applicationReportService = new ApplicationReportService(context);
+    	ApplicationReportModel applicationReportModel = applicationReportService.create();
+    	
+    	applicationReportModel.setReportPriority(10000);
+        applicationReportModel.setDisplayInApplicationReportIndex(true);
+        applicationReportModel.setReportName(REPORT_NAME);
+        applicationReportModel.setReportIconClass("glyphicon glyphicon-info-sign");
+        applicationReportModel.setMainApplicationReport(true);
+        applicationReportModel.setProjectModel(projectModel);
+        applicationReportModel.setTemplatePath(TEMPLATE_APPLICATION_REPORT);
+        applicationReportModel.setTemplateType(TemplateType.FREEMARKER);
+
+        Map<String, WindupVertexFrame> related = new HashMap<String, WindupVertexFrame>();
+        AboutWindupModel aboutWindupModel = context.getFramed().addVertex(null, AboutWindupModel.class);
+        
+        //TODO: replace with the utility call that @lincolnthree is going to be adding to the utility package.
+        aboutWindupModel.setWindupRuntimeVersion("2.3.0.SNAPSHOT");
```

Comment:
Not right now; @lincolnthree needs to expose the "determine version" utility out of the existing package to util.  Once he merges that in, we can replace the static value with his utility.  Right now, it is in the Bootstrap class, which isn't visible to this module.